$MirOS$

Fix their idea of COFF to be in sync
with <sys/exec_ecoff.h> (our kernel)
mostly done in asm.c but here I must
adjust the offsets

--- utils/8l/obj.c.orig	Fri Mar 31 01:38:42 2006
+++ utils/8l/obj.c	Fri Mar 31 03:14:10 2006
@@ -13,7 +13,7 @@ char	*thestring 	= "386";
 
 /*
  *	-H0 -T0x40004C -D0x10000000	is garbage unix
- *	-H1 -T0xd0 -R4			is unix coff
+ *	-H1 -T0xd4 -R4			is unix coff
  *	-H2 -T4128 -R4096		is plan9 format
  *	-H3 -Tx -Rx			is MS-DOS .COM
  *	-H4 -Tx -Rx			is fake MS-DOS .EXE
@@ -41,7 +41,7 @@ isobjfile(char *f)
 	return v;
 }
 
-void
+int
 main(int argc, char *argv[])
 {
 	int i, c;
@@ -134,9 +134,9 @@ main(int argc, char *argv[])
 			INITRND = 0;
 		break;
 	case 1:	/* is unix coff */
-		HEADR = 0xd0L;
+		HEADR = 0xd4L;
 		if(INITTEXT == -1)
-			INITTEXT = 0xd0;
+			INITTEXT = 0xd4;
 		if(INITDAT == -1)
 			INITDAT = 0x400000;
 		if(INITRND == -1)
